functions.php

```diff
@@ -6,7 +6,35 @@
  * Author URI:	    http://inpsyde.com/
  */
 
-add_action( 'after_setup_theme', 'kiel_child_starter_setup' );
+
+/**
+ * The .min suffix for stylesheets and scripts.
+ *
+ * In order to provide a quick start, this child theme by default will load
+ * regular CSS and javascript files (whereas its parent theme Hamburg loads
+ * minified versions of its stylesheets and scripts by default).
+ *
+ * If you want your child theme to default on minified stylesheets as well,
+ * just uncomment the line where SCRIPT_DEBUG is defined.
+ * Don’t forget to actually add applicable .min files to your child theme first!
+ *
+ * You can then temporarily switch back to unminified versions of the same
+ * files by setting the constant SCRIPT_DEBUG to TRUE in your wp-config.php:
+ * define( 'SCRIPT_DEBUG', TRUE );
+ *
+ * @since   02/09/2015
+ * @param   string $suffix (default: '')
+ * @return  string
+ */
+function kiel_child_starter_get_script_suffix( $suffix = '' ) {
+
+	// Uncomment to load minified stylesheet by default
+	// $suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+
+	return $suffix;
```

Comment:
Bitte in ändern:

```php
return apply_filters( 'kiel_child_starter_get_script_suffix', $suffix );
```

Wäre noch cooler, weil wir dadurch noch Plugins anbandeln lassen.